internal/daysteps/daysteps.go

```diff
@@ -12,9 +18,72 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	// 1. разделить строку на слайс строк
+	slice := strings.Split(data, ",")
+
+	// 2. проверить, чтобы длина слайса была равна 2, так как в строке данных у нас количество шагов и продолжительность
+	if len(slice) != 2 {
+		return 0, 0, fmt.Errorf("длина слайса не равна 2")
+	}
+
+	// 3. преобразовать первый элемент слайса (количество шагов) в тип int. Обработать возможные ошибки. При их возникновении
+	// из функции вернуть 0 шагов, 0 продолжительность и ошибку
+	steps, err := strconv.Atoi(slice[0])
+	if err != nil {
+		return 0, 0, fmt.Errorf("не удалось преобразовать количество шагов: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return 0, 0, fmt.Errorf("не удалось преобразовать количество шагов: %w", err)
```